src/Aquifer.Data/Entities/BibleTextEntity.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Aquifer.Data.Enums;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Data.Entities;
+
+[Index(nameof(BibleId), nameof(BookId), nameof(ChapterNumber), nameof(VerseNumber))]
+public class BibleTextEntity
+{
+    public int Id { get; set; }
+    public int BibleId { get; set; }
+    public BookId BookId { get; set; }
+    public int ChapterNumber { get; set; }
```

Comment:
Can this just be `BookChapterId`?

Reply:
I went with that initially but the problem is versification. We would end up tying the wrong things together. For instance the Psalms in Russian have most chapters offset by 1. This way we end up with the accurate Book Chapter Verse for each Bible and can handle versification later.

Reply:
Okay, that makes sense. Good thinking.